### PR TITLE
fix test_docstrings

### PR DESCRIPTION
**Pull request by xoiga123**
Consider [this problematic docstring](https://github.com/unifyai/ivy/blob/0fe675920f387d7052e0114b49ccca470ff7181d/ivy/functional/ivy/experimental/layers.py#L119):
```python
>>> x = ivy.arange(12.).reshape((2, 1, 3, 2))
>>> print(ivy.max_pool2d(x, (2, 2), (1, 1), 'SAME'))
    ivy.array([[[[ 2,  3],
             [ 4,  5],
             [ 4,  5]]],


           [[[ 8,  9],
             [10, 11],
             [10, 11]]]])

>>> x = ivy.arange(48.).reshape((2, 4, 3, 2))
>>> print(ivy.max_pool2d(x, 3, 1, 'VALID'))
    ivy.array([[[[16, 17]],

            [[22, 23]]],


           [[[40, 41]],

            [[46, 47]]]])
```

607cf5cb87829bf5c9904b6c88d5edcedd32f0b3 fails to detect the second example's input.
8a077d9b59365b0ae571737991c933a37e51a428 fails to detect the second example's output.
=> Only care about the first example.

Consider [this different example](https://github.com/unifyai/ivy/blob/0fe675920f387d7052e0114b49ccca470ff7181d/ivy/array/layers.py#L487):
```python
>>> x = ivy.array([[[[1.], [2.0],[3.]],
...                 [[1.], [2.0],[3.]],
...                 [[1.], [2.0],[3.]]]]) #NHWC
>>> filters = ivy.array([[[[0.]], [[1.]], [[0.]]],
...                      [[[0.]], [[1.]], [[0.]]],
...                      [[[0.]], [[1.]], [[0.]]]]) #HWIO
>>> result = x.conv2d(filters, 1, 'SAME', data_format='NHWC',
...    dilations= 1)
>>> print(result)
ivy.array([[
          [[2.],[4.],[6.]],
          [[3.],[6.],[9.]],
          [[2.],[4.],[6.]]
          ]])
```

8a077d9b59365b0ae571737991c933a37e51a428 even fails to detect the first example's output and returns true immediately without running the input!

### CODE REVIEW

**Comment by xoiga123** [comment]
Btw, I've re-ran `ivy_tests.test_docstrings.test_docstrings("torch")` with this PR and found many invalid docstrings (`map_nest_at_indices` and `std` didn't even bother to run and copy-paste the output :smile:)

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by xoiga123** [comment]
@ZoeCD hi, can you review this please 😃

**Comment by danielmunioz** [comment]
Hi! Thanks for the contribution, Good Catch! I believe the `continue` on line 134 and the `print` comments on 190-191 may no longer be necessary? but feel free to correct me, other than that LGTM! @ZoeCD

File: ivy_tests/test_docstrings.py [hunk 1]
```diff
@@ -105,10 +103,14 @@ def check_docstring_examples_run(
     sub = ">>> print("
     for index, line in enumerate(trimmed_docstring):
         if sub in line:
+            # find chunks of output
```
[the remark this answers is not in the document]
Comment (xoiga123) [reply ivy_tests/test_docstrings.py]: Got it

**Comment by xoiga123** [comment]
@ZoeCD hi, in case you forgot

**Pull request merged**